src/components/Snackbar/Snackbar.tsx

```diff
@@ -27,7 +27,7 @@ const Snackbar: React.FC<SnackbarProps> = ({ destroy, duration = 6000, ...rest }
     return () => clearTimeout(timeoutRef.current as number);
   }, [duration]);
 
-  return <ControlledAlert {...rest} open onClose={destroy} discardable />;
```

Comment:
why? By doing that you can't press "X" on the snackbar

Reply:
This is by design. @dimitriszoitas  asked me to remove this.

Reply:
Plus we can always add the `discardable` prop in the `snackbar` component itself if we ever need to! `Snackbar` passes this prop to `ControlerAlert`.

Reply:
sure!